Sources/StreamVideoSwiftUI/CallView/ScreenSharing/ScreenSharingView.swift

```diff
@@ -24,7 +24,24 @@ public struct ScreenSharingView<Factory: ViewFactory>: View {
         screenSharing: ScreenSharingSession,
         availableFrame: CGRect,
         innerItemSpace: CGFloat = 8,
-        viewFactory: Factory = DefaultViewFactory.shared,
+        isZoomEnabled: Bool = true
+    ) where Factory == DefaultViewFactory {
```

Comment:
Weird that we have to solve it with 2 methods now, adding `@MainActor` to the init doesn't help as well.

Reply:
Access to static properties became very weird on Xcode 15.3 :/